src/web/src/auth.ecpp

```diff
@@ -53,8 +53,8 @@ std::string checked_token;
     std::string token = qparam.param ("token");
     std::string audit_msg;
     if (request.getMethod () == "POST")
-        audit_msg = std::string ("Request CREATE auth username ") + username + " FAILED";
-    check_regex_text_or_die ("token", token, checked_token, "^[-/._~+/a-z0-9]{0,255}={0,10}$",
+        audit_msg = std::string ("Request CREATE auth FAILED");
+    check_regex_text_or_die_audit ("token", token, checked_token, "^[-/._~+/a-z0-9]{0,255}={0,10}$",
```

Comment:
For future: macro names should be all same-style, commonly upper-cased.

Also, the `CHECK_USER_PERMISSIONS_OR_DIE_AUDIT` above casts `(char *) nullptr` while other cases I've seen recently do not...